projects/m5/003-probability-calculator/js/prob_calculator.js

```diff
@@ -0,0 +1,58 @@
+const cloneDeep = require("lodash.clonedeep");
+
+class Hat {
+  constructor(balls = { black: 1 }) {
```

Comment:
Here you accept only one parameter with multiple elements, 
not a variable number of arguments, as required by the trace.
What could be a workaround?